static/examples/cstg-secure-signals-example/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</title>
+    <link rel="stylesheet" type="text/css" href="./styles/ads.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/app.css" />
+    <link rel="shortcut icon" href="images/favicon.png" />
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+
+    </script>
+  </head>
+  <body>
+    <h1>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the
+      <a href="https://unifiedid.com/docs/guides/publisher-client-side"
+        >Client-Side Integration Guide for JavaScript
+      </a>
+      to implement UID2 integration and generate UID2 tokens.
+      Secure Signals is updated when the page is reloaded.  
+      Reload the page when logging in order out to update Secure Signals in local storage.
```

Comment:
"Reload the page when logging in order out to update..."
Not sure exactly what this should say. Maybe "in order to update"?
Not sure about the logging, prob fine? Or does it mean the user logging in? We've removed a lot of mentions of user login.

Reply:
sorry.  changed to "Reload the page in order to update Secure Signals in local storage."